### PR TITLE
Feature/ch203/create epic in clubhouse went a clickup task

### PR DESCRIPTION
**Pull request by theantichris**
This PR has the functionality to create an Epic on Clubhosue when a ClickUp Task is moved to Ready for Development.

I have created another [story](https://app.clubhouse.io/aoeu-se/story/227/move-clubhouse-and-clickup-functions-into-packages) to extract the ClickUp and Clubhouse API functionality to separate packages and will include test coverage with that once it is easier.

### CODE REVIEW

**Review by Thomas-Jean — DISMISSED** [review]

File: DECISION-LOG-BOOK.md [hunk 1]
```diff
@@ -4,70 +4,40 @@ Taken from [Decision Management in Software Engineering](https://medium.com/swlh
...
 * Christopher Lamm
+* Chris Poulton
+* Bob Yexley
 * Thomas Jean
+* Judy Mosley
 
 ### Context
 
-We will be using Go for several services and need to pick an ORM to use across them all, so we have uniformity.
+We need to be able to manager and work stories through our normal process in Clubhouse while allowing Chris to manage the product side in ClickUp.
```
Comment (ryexley) [inline DECISION-LOG-BOOK.md:19]: "manager" -> "manage", right?

File: DECISION-LOG-BOOK.md [hunk 2]
```diff
@@ -4,70 +4,40 @@ Taken from [Decision Management in Software Engineering](https://medium.com/swlh
...
 
 ### Solution
 
-[ent](https://github.com/facebook/ent)
+Developer a service that will automatically updated ClickUp and Clubhouse based on events in the opposite tool.
```
Comment (ryexley) [inline DECISION-LOG-BOOK.md:23]: "Developer" -> "Develop", right?

**Review by ryexley — APPROVED** [review]
This will be super nice to have 👍🏼

**Review by ryexley — APPROVED** [review]

**Review by ryexley — APPROVED** [review]

**Review by Thomas-Jean — APPROVED** [review]

File: DECISION-LOG-BOOK.md [hunk 3]
```diff
@@ -4,70 +4,40 @@ Taken from [Decision Management in Software Engineering](https://medium.com/swlh
...
 
 ### Context
 
-We will be using Go for several services and need to pick an ORM to use across them all, so we have uniformity.
+We need to be able to manage and work stories through our normal process in Clubhouse while allowing Chris to manage the product side in ClickUp.
 
 ### Solution
 
-[ent](https://github.com/facebook/ent)
+Develop a service that will automatically updated ClickUp and Clubhouse based on events in the opposite tool.
```
Comment (jmosley5) [inline DECISION-LOG-BOOK.md:23]: Change "updated" to "update"

**Pull request merged**